docs/examples/extensions/table-column/js/index.js

```diff
@@ -7,7 +7,7 @@ import { addFilter } from '@wordpress/hooks';
 import { Rating } from '@woocommerce/components';
 
 addFilter( 'woocommerce_admin_report_table', 'plugin-domain', reportTableData => {
-	if ( 'products' !== reportTableData.endpoint || ! reportTableData.items.data.length ) {
+	if ( 'products' !== reportTableData.endpoint || ! reportTableData.items || ! reportTableData.items.data.length ) {
```

Comment:
should we be extra paranoid and add a ` || report.TableData.items.data ` too?

Reply:
Extra paranoia added! https://github.com/woocommerce/woocommerce-admin/pull/2488/commits/da7c70f599612354ccedfbf0591c66b597560334